### PR TITLE
chore: dedupe regression test server start code

### PR DESCRIPTION
**Pull request by jhult**
Resolves Granola-Team/mina-indexer#778

### CODE REVIEW

File: tests/regression [hunk 1]
```diff
@@ -95,6 +95,13 @@ idxr_server_start() {
     idxr_server start --web-port "$port" "$@"
 }
 
+idxr_server_start_standard() {
```
Comment (robinbb) [inline tests/regression:98]: Function needs to take further arguments that are appended to the command via "$@"

**Review by robinbb — DISMISSED** [review]

File: tests/regression [hunk 2]
```diff
@@ -301,10 +305,7 @@ test_ipc_is_available_immediately() {
     setup
     dl_mainnet 100 ./blocks
 
-    idxr_server_start \
-        --blocks-dir ./blocks \
-        --staking-ledgers-dir ./staking-ledgers \
-        --database-dir ./database \
+    idxr_server_start_standard \
```
[the remark this answers is not in the document]
Comment (jhult) [reply tests/regression:309]: Fixed.

File: tests/regression (hunk 1 above)
Comment (jhult) [reply tests/regression:98]: Fixed.

**Review by Isaac-DeFrain — APPROVED** [review]
LGTM

**Review by Isaac-DeFrain — APPROVED** [review]
LGTM

**Pull request merged**